### DISCUSSION TITLE
How to use with Mixed chart types

### DISCUSSION THREAD

**Discussion by toniengelhardt** [victorgarciaesgi/vue-chart-3#106] [General]
How would one implement a mixed chart, e.g. Line + Bar, there is no MixedChart export?

**Update:**

The following seems to work:
 
```ts
const chartData = computed<ChartData<any>>(() => ({
  ...
})
```

But to get correct typing should it not be like this:

```ts
const chartData = computed<ChartData<"bar" | "line">>(() => ({
  ...
})
```

or this:

```ts
const chartData = computed<ChartData<"mixed">>(() => ({
  ...
})
```

**Comment by victorgarciaesgi** [comment]
It's a limitation of the `ChartData` type from Chart.js. I will see if a can make a `MixedData` type. And i need to make a section for mixed charts in the docs too

**Reply by toniengelhardt** [reply]
Great, thanks! If it is not possible, a small note in the docs saying that type `any` works would be very useful I think.
